### PR TITLE
[DOCS] Add Windows and Firefox mkcert install instructions

### PR DESCRIPTION
**Pull request by gilbertsoft**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -101,6 +101,26 @@ On variants of Linux you may be prompted for additional package installation to
...
 
+### Windows and Firefox `mkcert -install` additional instructions
+
+The `mkcert -install` step on Windows does not work for the Firefox browser.
+You need to add the created root certficate authority to the security
+configuration by your self:
+
+- Run `mkcert -install` (you can use the shortcut from the start menu for that)
+- Run `mkcert -CAROOT` to see the local folder used for the newly created root
+  certificate authority before
```
Comment (rfay) [inline docs/index.md]: "before" is extraneous.

File: docs/index.md [hunk 2]
```diff
@@ -101,6 +101,26 @@ On variants of Linux you may be prompted for additional package installation to
...
+- Open the settings
```
Comment (rfay) [inline docs/index.md]: Firefox settings? What settings?

File: docs/index.md (hunk 1 above)
Comment (gilbertsoft) [reply docs/index.md]: done

File: docs/index.md (hunk 2 above)
Comment (gilbertsoft) [reply docs/index.md]: done

**Pull request merged**